clio-server/src/main/scala/org/broadinstitute/clio/server/service/ServerService.scala

```diff
@@ -64,9 +64,9 @@ class ServerService private (
 
       persistenceDAO
         .getAllSince(latestUpsert)
-        .runWith(Sink.foldAsync(0) { (count, json) =>
-          logger.debug(s"Recovering document with ID ${getUpsertId(json)}")
```

Comment:
I deleted this log because elastic4s already logs out the full JSON body of the POST when set to debug logging, so this doesn't buy us much.